third_party/openthread/CMakeLists.txt

```diff
@@ -76,6 +76,10 @@ if (OTBR_SRP_ADVERTISING_PROXY)
     set(OT_EXTERNAL_HEAP ON CACHE STRING "enable external heap" FORCE)
 endif()
 
+if (OTBR_EPSKC)
+    set(OT_BORDER_AGENT_EPSKC ON CACHE STRING "enable border agent ephemeral PSKc" FORCE)
+endif()
```

Comment:
Let's try to make features runtime configurable instead of build time.

Reply:
+1 on the suggestion

> Let's try to make features runtime configurable instead of build time.

I don't see the necessity of doing it in this PR. Could you elaborate?

Reply:
the runtime configuration api doesn't need to be in this PR.

Reply:
Accepted. Thanks~~